doc/dnsseed-policy.md

```diff
@@ -43,7 +43,7 @@ related to the DNS seed operation.
 
 If these expectations cannot be satisfied the operator should
 discontinue providing services and contact the active Bitcoin
-Core development team as well as posting on
+Classic development team as well as posting on
```

Comment:
Should they contact the Classic team or the Core team, or both?
Should they post on the bitcoin-dev mailing list or somewhere else?